README.md

```diff
@@ -1,2 +1,16 @@
 # java-globaltracer
 Global ThreadLocal Span for implicit propagation, delegating to another Tracer implementation.
+
```

Comment:
Agreed!
Unfortunately, I have only had the time so far to (re-)write the code and a working unit test.
I hope to make some time this evening to provide examples in the README.